app/components/TutorialsCategory.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-defineProps<{
+const props = defineProps<{
```

Comment:
Props isn't used in the script portion
```suggestion
defineProps<{
```